src/pages/Detailed/MobileDetailedPage.js

```diff
@@ -24,29 +32,86 @@ const MobileDetailedPage = props => {
     </div>
   )
 
+  let transactionVolumeInfo
+  const { transactionVolume } = props
+  if (transactionVolume && transactionVolume.length === 2) {
+    const [yesterday, today] = transactionVolume
+    const TVDiff = calcPercentageChange(
+      yesterday.transactionVolume,
+      today.transactionVolume
+    )
+    if (today.transactionVolume > 0) {
+      transactionVolumeInfo = {
+        name: 'Transaction Volume',
+        value: today.transactionVolume,
+        label: '24h',
+        changes: TVDiff
+      }
+    }
+  }
+
+  let activeAddressesInfo
+  const { dailyActiveAddresses } = props
+  if (dailyActiveAddresses && dailyActiveAddresses.length === 2) {
+    const [yesterday, today] = dailyActiveAddresses
```

Comment:
`const [{activeAddresses: yesterdayActiveAddresses}, {activeAddresses: todayActiveAddresses}] = ...` (or some other variable alias)
Better to use destruction here, it will not affect code readability but will increase perfomance. Same for LOC `40`